Kernel/Bus/VirtIO/Device.cpp

```diff
@@ -46,42 +46,44 @@ UNMAP_AFTER_INIT void detect()
     });
 }
 
-StringView determine_device_class(const PCI::Address& address)
+static StringView const determine_device_class(const PCI::Address& address)
 {
     if (PCI::get_revision_id(address) == 0) {
         // Note: If the device is a legacy (or transitional) device, therefore,
         // probe the subsystem ID in the PCI header and figure out the
         auto subsystem_device_id = PCI::get_subsystem_id(address);
         switch (subsystem_device_id) {
         case 1:
-            return "VirtIONetAdapter";
+            return "VirtIONetAdapter"sv;
```

Comment:
I'd argue to apply these changes in a separate commit just to make it more "bisectable" :)

Reply:
done